frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -141,7 +147,7 @@ export const UpdateLink: React.SFC<CurrentVersionProps> = ({cv}) => {
   return <React.Fragment>
     {
       updatesAvailable && (status === ClusterUpdateStatus.ErrorRetrieving || status === ClusterUpdateStatus.Failing || status === ClusterUpdateStatus.Updating)
-        ? <Button bsStyle="link" className="btn-link--no-btn-default-values" onClick={() => (clusterUpdateModal({cv}))}>Update to another version</Button>
+        ? <Button variant="link" className="btn-link--no-btn-default-values" onClick={() => (clusterUpdateModal({cv}))}>Update to another version</Button>
```

Comment:
Should plan to rename `btn-link--no-btn-default-values` to PF4 classname when sweeping for `btn` on the next go round.